lib/google_drive/worksheet.rb

```diff
@@ -139,6 +139,19 @@ def human_url
       format("%s\#gid=%s", spreadsheet.human_url, gid)
     end
 
+    # copy worksheet to specified spreadsheet
+    def copy_to(spreadsheet_id)
```

Comment:
Can you let this method take an instance of GoogleDrive::Spreadsheet (maybe addition to allowing an ID)? That would be more natural API.

Reply:
This also worried me.
First I thought, official API reference says “Spreadsheets.sheets.copyTo”.
And the name “copy_to” could simply be taken as the destination_spreadsheet_key.
So, I decided to add this to GoogleDrive :: Worksheet.

However, file logic should be aggregated into GoogleDrive :: Spreadsheet, which inherits GoogleDrive :: File.

When moving this method to GoogleDrive :: Spreadsheet, what should I do with the arguments...?

This method takes `destination_spreadsheet_id` and `sheet_id`.
In my opinion, these two should be passed as just ordinary arguments (because they are needed)
The order is a bit annoying, but considering the method name, destination_spreadsheet_id will come first.

Sorry for the long time. We look forward to reply!!!

Reply:
It would be OK to keep this as a method of Worksheet because that's the primary object of the action.

I just suggested to let the method take a single Spreadsheet object instead of an ID:

```ruby
ss1 = session.spreadsheet_by_name("Spreadsheet 1")
ss2 = session.spreadsheet_by_name("Spreadsheet 2")
ss1.worksheets[0].copy_to(ss2)
```

It is possible that the user wants to directly pass an ID which they have got somewhere, so I'm OK accepting an ID too:
```ruby
ss1 = session.spreadsheet_by_name("Spreadsheet 1")
ss1.worksheets[0].copy_to("bbb")
```

Reply:
Oh! I was misunderstanding. I'll fix it.
Thank you carefully!